index.dd

```diff
@@ -93,30 +93,21 @@ $(DIVC boxes,
                 Adam D. Ruppe's weekly newsletter. $(SPANC twid)
             )
         )
-        $(TOUR comments, Community,
-            $(P Discuss D on the $(LINK2 https://forum.dlang.org/, forums), join
-                the $(LINK2 irc://irc.freenode.net/d, IRC channel), or follow
-                us on $(LINK2 https://twitter.com/D_Programming, Twitter).)
-            $(P Browse the $(LINK2 https://wiki.dlang.org, wiki) where among
-                other things you can find the
-                $(LINK2 https://wiki.dlang.org/Review_Queue, review queue) for
-                major additions to the D project.)
-        )
-    )
-    $(DIVC row,
         $(TOUR graduation-cap, Learn,
             $(P Quick start with the $(LINK2 http://tour.dlang.org,
-                introduction tour to D),
+                Introduction to D),
             	check out Ali Çehreli's great textbook
                 $(LINK2 http://ddili.org/ders/d.en/index.html,
                 <cite>Programming in D</cite>), ask questions in the
                 $(LINK2 https://forum.dlang.org/group/digitalmars.D.learn,
-                    Learn forum),
+                    Learn forum) and
```

Comment:
I like it better with just the comma. An "and" seems to say that you have to do all that. An "or" would seem to say that they're mutually exclusive. Just a comma is like an "and/or" between the elements. Or maybe I'm seeing a difference where there is none.

Anyway: Oxford comma.